### ISSUE TITLE
mongo-data use different image version?

### ISSUE THREAD

**Issue by katopz** [yongjhih/docker-parse-server#85]
Refer to https://github.com/yongjhih/docker-parse-server/blob/master/docker-compose.yml#L94
```yml
mongo:
  image: mongo:3.0.8
```
So `mongo-data` should use same image or it will pull new different one?
```yml
mongo-data:
  image: mongo
```

Thanks

**Comment by yongjhih** [comment]
@katopz ,
Yes, I noticed that before, but I'm afraid that modification affects somebody destroy his data. So I did not to change it. Though it should be changed.
I'm ready to merge your pull-request. Thanks.

**Comment by mzgnr** [comment]
hi, @yongjhih any progress on this ?
